### DISCUSSION TITLE
Cannot play video on some beatmap

### DISCUSSION THREAD

**Discussion by HelloYeew** [ppy/osu#15443] [Q&A]
Since last update (2021.1103.1) some beatmap's video cannot play due to the new video codec update.

https://user-images.githubusercontent.com/68165621/140002589-8862cdb7-daf6-4dac-aa70-9eef2c6d7019.mp4

Log:
[runtime.log](https://github.com/ppy/osu/files/7464992/runtime.log)

Beatmap:
[First beatmap](https://osu.ppy.sh/beatmapsets/479107#osu/1023083)
[Second beatmap (that have video problem)](https://osu.ppy.sh/beatmapsets/1120850#osu/2644964)

**Comment by frenzibyte** [comment]
To clarify, this gets resolved by disabling hardware acceleration, correct?

**Reply by HelloYeew** [reply]
I turned off the hardware and the video is still not playing.

https://user-images.githubusercontent.com/68165621/140004065-41e5b33e-5730-430b-9a25-62fc867bc55c.mp4

Log:
[runtime.log](https://github.com/ppy/osu/files/7465028/runtime.log)

**Reply by peppy** [reply]
Can you confirm that other beatmaps do successfully play video?

**Reply by HelloYeew** [reply]
I try to play on other beatmap and some beatmap have other like this too. I have collected some of other beatmaps that have this problem for you.

Screen record when I try on other beatmap:



https://user-images.githubusercontent.com/68165621/140014051-25374f75-c4ac-458a-ab84-1c95df37db3f.mp4


Log:
[runtime.log](https://github.com/ppy/osu/files/7465517/runtime.log)

Beatmap in list:
[ADAMAS](https://osu.ppy.sh/beatmapsets/859608#osu/1799255)
[unravel](https://osu.ppy.sh/beatmapsets/201141#osu/476367)
[Yubi Bouenkyou](https://osu.ppy.sh/beatmapsets/1178935#osu/2470378)
[courage](https://osu.ppy.sh/beatmapsets/226289#osu/527709)

**Reply by peppy** [reply]
My question was more: does *any* video work for you?

**Reply by HelloYeew** [reply]
More than I list it's worked.

**Reply by peppy** [reply]
Thanks, we're tracking this over at the framework repository: https://github.com/ppy/osu-framework/issues/4858
